guix/README.md

```diff
@@ -0,0 +1,79 @@
+# Jupyter Notebook Dependencies
+
+Although GitHub renders Jupyter notebooks, in order to be able to use the
+interactive portions or to run/modify the code yourself, you will need the
+python dependencies.
```

Comment:
Thanks, much better. For example I wasn't aware of this crucial fact:

> GitHub renders Jupyter notebooks

With this in mind, can you create another file for this setup and leave only the link to the `Amount Org.ipynb` in the README?

Reply:
you mean merge the `guix/README.md` file into the existing `README`? maybe more organized would be to put the notebook stuff in a separate subdirectory? i didn't really think about the layout but in hindsight it was probably a mistake to clutter up the root directory